src/category_theory/adjunction/reflective.lean

```diff
@@ -102,11 +103,53 @@ begin
   exact mem_ess_image_of_unit_is_iso A,
 end
 
-universes v₃ u₃
-variables {E : Type u₃} [category.{v₃} E]
-
 /-- Composition of reflective functors. -/
 instance reflective.comp (F : C ⥤ D) (G : D ⥤ E) [Fr : reflective F] [Gr : reflective G] :
   reflective (F ⋙ G) := { to_faithful := faithful.comp F G, }
 
+/-- (Implementation) Auxiliary definition for `unit_comp_partial_bijective`. -/
+def unit_comp_partial_bijective_aux [reflective i] (A : C) (B : D) :
+  (A ⟶ i.obj B) ≃ (i.obj ((left_adjoint i).obj A) ⟶ i.obj B) :=
+((adjunction.of_right_adjoint i).hom_equiv _ _).symm.trans (equiv_of_fully_faithful i)
+
+/-- The description of the inverse of the bijection `unit_comp_partial_bijective_aux`. -/
+lemma unit_comp_partial_bijective_aux_symm_apply [reflective i] {A : C} {B : D}
+  (f : i.obj ((left_adjoint i).obj A) ⟶ i.obj B) :
+  (unit_comp_partial_bijective_aux _ _).symm f = (of_right_adjoint i).unit.app A ≫ f :=
+by simp [unit_comp_partial_bijective_aux]
+
+/--
+If `i` has a reflector `L`, then the function `(i L A ⟶ B) → (A ⟶ B)` given by precomposing with
```

Comment:
I know it's tedious, but I think it's best if code inside backticks acts like Lean code: so probably we want `(i.obj (L.obj A) ⟶ B)` here. Otherwise my internal typechecker starts complaining. :-)

Reply:
The alternative is to switch to $$ and write normal maths.

Reply:
Amended, thanks